### ISSUE TITLE
VG RDF isn't vg:offset a duplicative value of vg:rank 

### ISSUE THREAD

**Issue by JervenBolleman** [vgteam/vg#2497]
During the discussions with @subwaystation and @josiahseaman I realized that the rank value is duplicative in it's content with the offset value.

`vg:offset`s are `vg:rank` in nucleotide space. A sort of steps by rank or a sort of steps by offset must give the same value. As the only purpose `vg:rank` is to be able to such a sort it should be left out of the RDF serialization as it's purpose is served by `vg:offset`. 

Or am I missing some query cases that depend on `vg:rank` being monotonically increasing along the start order.

**Comment by josiahseaman** [comment]
Agreed.  I would also add to this that we should keep the nucleotide resolution integer, as it is more versatile and harder to calculate.  I think the number of nodes in the path is less useful.

**Comment by subwaystation** [comment]
As soon as I am deeper into the RDF world, I will investigate this further in detail. I am also open to adjust the `C++` code, if necessary.

@JervenBolleman I am wondering which **throw out** would give us the most efficient solution in terms of computation and disk storage including I/O. Or does this not matter in the end, when we have the whole thing in a SPARQL endpoint?
